merlin/dag/base_operator.py

```diff
@@ -409,3 +410,92 @@ def _get_columns(self, df, selector):
             return {col_name: df[col_name] for col_name in selector.names}
         else:
             return df[selector.names]
+
+    @property
+    def export_name(self):
+        """
+        Provides a clear common english identifier for this operator.
+
+        Returns
+        -------
+        String
+            Name of the current class as spelled in module.
+        """
+        return self.__class__.__name__.lower()
+
+    def export(self, path: str, input_schema: Schema, output_schema: Schema, **kwargs):
+        """
+        Export the class object as a config and all related files to the user defined path.
+
+        Parameters
+        ----------
+        path : str
+            Artifact export path
+        input_schema : Schema
+            A schema with information about the inputs to this operator.
+        output_schema : Schema
+            A schema with information about the outputs of this operator.
+        params : dict, optional
+            Parameters dictionary of key, value pairs stored in exported config, by default None.
+        node_id : int, optional
+            The placement of the node in the graph (starts at 1), by default None.
+        version : int, optional
+            The version of the model, by default 1.
+
+        Returns
+        -------
+        model_config: dict
+            The config for the exported operator (Triton model).
+        """
+        raise NotImplementedError(
+            "Exporting an operator to run in a particular context (i.e. Triton) "
+            "only makes sense when a runtime is specified. To select an "
+            f"operator for the appropriate runtime, replace {self.__class__.__name__} "
+            f"with a runtime-specific operator class, possibly {self.__class__.__name__}Triton"
+        )
+
+    @classmethod
+    def from_model_registry(
```

Comment:
I don't think I'm ready to say that model registry integration should be part of Merlin's core functionality. It sort of makes sense in Systems, but even there I wonder how directly it needs to be integrated into operator classes. If we're going to put an `export()` method in Core, then I do think it makes sense to have a corresponding method that loads it back from the same path (i.e. `from_path` or some variant of it), but this convenience method adds unnecessary coupling between operators and model registries.